src/app/models/AudioRecording.ts

```diff
@@ -113,7 +116,11 @@ export class AudioRecording
   }
 
   public get downloadUrl(): string {
-    // TODO Add download url when download page built
+    return downloadAudioRecordingMenuItem.uri({ audioRecordingId: this.id });
```

Comment:
seems like we should be pointing to a strongroute instance?

Reply:
The original download route is an external route to the api server. StrongRoute is for internal links only. Unless im missing something